src/Npgsql/TypeMapping/ConnectorTypeMapper.cs

```diff
@@ -38,6 +40,7 @@ internal NpgsqlDatabaseInfo DatabaseInfo
 
             _databaseInfo = value;
             Reset();
+            _pgVersion = value.Version;
```

Comment:
Is there a specific reason why we can't just use the version from the `DatabaseInfo`?

Reply:
Just thought to cache the value, but I guess you're right - changed.